<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionWindow</class>
 <widget class="QWidget" name="ActionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableWidget" name="windowTableWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="columnCount">
      <number>0</number>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QTableWidget" name="controlsTableWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="allWindowsButton">
     <property name="text">
      <string>Get All Windows</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="controlsButton">
     <property name="text">
      <string>Get controls for current window</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Action</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rbCombo">
        <property name="text">
         <string>Combo box</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbButton">
        <property name="text">
         <string>Button</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbEdit">
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTextEdit" name="infoTextEdit"/>
   </item>
   <item row="3" column="1">
    <widget class="QTextEdit" name="setterTextEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="infoButton">
     <property name="text">
      <string>Get more info</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="textButton">
     <property name="text">
      <string>Action</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLineEdit" name="processLineEdit"/>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="processButton">
     <property name="text">
      <string>Start process</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
